journal/updated/01-journal.adoc

```diff
@@ -1,146 +1,152 @@
-as too many are climbing up that way, which is not the way to God the Father;
-the way to the Father is the way of holiness and purity, and humility,
-without which no man shall see the Lord nor enjoy his presence to their comfort.
-
-About this time, being in the year 1656, our ministers told us,
-that there was a sort of people come up in the north, called Quakers,
-that were a people of a strange posture and principles: saying,
-that it was the last days and times that Christ spoke of in the 24th of Matthew,
-"`Many shall come in my name, and deceive many, ver. 5.
+to seek after those things that are too high for them;
+too many are climbing up in that way, which is not the way to God the Father.
+The way to the Father is the way of holiness and purity, and humility,
+without which no man shall see the Lord, nor enjoy His presence to their comfort.
+
+About this time, being in the year 1656, our ministers told us
+that there was a sort of people emerging in the north, called Quakers,
+that were a people of strange practice and principles, saying
+that it was the last days and times that Christ spoke of in the 24th of Matthew:
+"`Many shall come in My name, and deceive many,
 for there shall arise false Christs, and false prophets,
 and shall show great signs and wonders, insomuch that if it were possible,
 they shall deceive the very elect.`"
-This sort of people called Quakers, were much preached against;
-they told us they were the false prophets, etc. that they denied the scriptures,
+This sort of people called Quakers were much preached against;
+our ministers told us they were the false prophets, etc., that they denied the scriptures,
 and all ordinances, and also denied the very Christ that bought them.
 They were represented to us to be such a dangerous sort of people,
 that we were afraid of any who had the name of a Quaker,
 lest we should be deceived by them.
 Hitherto they had not been in these parts of the country,
-neither did we know what were the principles held out by themselves;
-but only such as were reported, though falsely, unto us by our preachers and others;
-which kept us in blindness, and from making further inquiry, and "`trying all things,
-and holding fast that which is good,`" according to the apostle`'s advice, 1 Thess. 5:21.
-
-Now about the year 1657, there came a poor man in a mean habit to my master`'s house,
-named Morgan Evan, of South Wales:
-he had met with the people called Quakers in his travels, and was convinced of the truth.
+nor did we know what principles they adhered to,
+but only such as were reported, though falsely, unto us by our preachers and others.
+This kept us in blindness, and from making further inquiry, by
+"`trying all things, and holding fast that which is good,`"
+according to the apostle`'s advice, 1 Thess. 5:21.
+
+Now about the year 1657, there came a poor man dressed
+in drab apparel to my master`'s house named Morgan Evan, of South Wales.
```

Comment:
Now that I know more of the context, let's change "drab apparel" to "plain apparel". I would imagine that "mean habit" here is another way of saying plain dress.